src/main/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/CreateCollectionCommandResolver.java

```diff
@@ -164,6 +164,13 @@ private CreateCollectionCommand.Options.VectorSearchConfig validateVectorOptions
           ErrorCode.VECTOR_SEARCH_NOT_AVAILABLE,
           ErrorCode.VECTOR_SEARCH_NOT_AVAILABLE.getMessage());
     }
+
+    if (vector.vectorizeConfig() != null && !operationsConfig.vectorizeEnabled()) {
+      throw new JsonApiException(
+          ErrorCode.VECTORIZE_FEATURE_NOT_AVAILABLE,
+          ErrorCode.VECTORIZE_FEATURE_NOT_AVAILABLE.getMessage());
+    }
+
```

Comment:
Will be better to change to our new error style `throw ErrorCode.VECTORIZE_FEATURE_NOT_AVAILABLE.toApiException();`